e2e_tests/src/e2e_indexer/e2elive.py

```diff
@@ -151,17 +108,7 @@ def main():
     try:
         logger.info("reached expected round={}".format(lastblock))
         if not args.read_only:
-            xrun(
-                [
-                    "validate-accounting",
-                    "--verbose",
-                    "--algod",
-                    algoddir,
-                    "--indexer",
-                    indexerurl,
-                ],
-                timeout=20,
```

Comment:
I'm not so sure this timeout was even guaranteed to engage, see https://stackoverflow.com/questions/36952245/subprocess-timeout-failure

Reply:
Added this observation as an item on #1525 
<img width="815" alt="image" src="https://github.com/algorand/indexer/assets/291133/5737f578-c6f6-4bc9-9489-687a33fd35ef">